### ISSUE TITLE
Qwen3-Coder-480B-A35B-Instruct q4km has accuracy  issues

### ISSUE THREAD

**Issue by wenhuach21** [intel/auto-round#690]
may related to zeros in imatrix

**Comment by wenhuach21** [comment]
this is fine 
~~~python
import torch
from transformers import AutoModelForCausalLM, AutoTokenizer
from auto_round import AutoRound

model_name = "/data2/Qwen3-Coder-480B-A35B-Instruct"

model = AutoModelForCausalLM.from_pretrained(model_name,
                                             device_map="cpu", torch_dtype="auto")
tokenizer = AutoTokenizer.from_pretrained(model_name)
layer_config = {}
for n, m in model.named_modules():
    if n == "lm_head" or isinstance(m, torch.nn.Embedding):
        layer_config[n] = {"bits": 8}
    elif isinstance(m, torch.nn.Linear) :
        layer_config[n] = {"bits": 4}
autoround = AutoRound(model, tokenizer, iters=0, nsamples=512, dataset="github-code-clean",disable_opt_rtn=True,layer_config=layer_config)
autoround.quantize_and_save("/dataset/Qwen3-Coder-480B-A35B-Instruct-q4km", format="gguf:q4_k_s")
exit()
~~~
